8.1.3 - Issue with running program when zx is webpacked due to deno.js

Hello,

I noticed an issue with zx version 8.1.3 and newer, where zx does not work well with Webpack. The problem occurs when the program tries to load deno.js, which does not exist after bundling the application. 8.1.2 seems to work fine, although reference to `node_modules/zx/build/deno.js` is the output file since 8.1.0.

```sh
node:internal/modules/cjs/loader:1249
  const err = new Error(message);
              ^

Error: Cannot find module './index.cjs'
Require stack:
- /Users/krzysiek/Documents/work/dev/td1023/gnss/node_modules/zx/build/deno.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Function._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:141:16)
    at 13103 (/usr/jrd/gnss/vendor.js:1:1658751)
    at u (/usr/jrd/gnss/reset.js:1:6143)
    at /usr/jrd/gnss/reset.js:1:3399
    at u.a (/usr/jrd/gnss/reset.js:1:6736) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/krzysiek/Documents/work/dev/td1023/gnss/node_modules/zx/build/deno.js'
  ]
}

Node.js v22.11.0
```

![Image](https://github.com/user-attachments/assets/f8734a22-abef-4d51-9f77-ff1f8eeb57c5)

tsconfig:
```json
{
  "compilerOptions": {
    "module": "ESNext",
    "moduleResolution": "Bundler",
    "target": "ES2023",
    "lib": ["ES2023"],
    "outDir": "./build",
    "baseUrl": ".",
  }
}
```

--- a/package.json
+++ b/package.json
@@ -107,7 +107,7 @@
     "esbuild-node-externals": "^1.15.0",
     "esbuild-plugin-entry-chunks": "^0.1.15",
     "esbuild-plugin-extract-helpers": "^0.0.6",
-    "esbuild-plugin-hybrid-export": "^0.3.0",
+    "esbuild-plugin-hybrid-export": "^0.3.1",
     "esbuild-plugin-resolve": "^2.0.0",
     "esbuild-plugin-transform-hook": "^0.1.1",
     "esbuild-plugin-utils": "^0.1.0",

--- a/package.json
+++ b/package.json
@@ -107,7 +107,7 @@
     "esbuild-node-externals": "^1.15.0",
     "esbuild-plugin-entry-chunks": "^0.1.15",
     "esbuild-plugin-extract-helpers": "^0.0.6",
-    "esbuild-plugin-hybrid-export": "^0.2.5",
+    "esbuild-plugin-hybrid-export": "^0.3.0",
     "esbuild-plugin-resolve": "^2.0.0",
     "esbuild-plugin-transform-hook": "^0.1.1",
     "esbuild-plugin-utils": "^0.1.0",

--- a/scripts/build-js.mjs
+++ b/scripts/build-js.mjs
@@ -85,7 +85,7 @@ if (bundle === 'src') {
 if (hybrid) {
   plugins.push(
     hybridExportPlugin({
-      loader: 'require',
+      loader: 'reexport',
       to: 'build',
       toExt: '.js',
     })
@@ -100,7 +100,7 @@ plugins.push(
         if: !hybrid,
         pattern: /\.js$/,
         transform(contents) {
-          return injectCode(contents, `import { require } from './deno.js'`)
+          return injectCode(contents, `import './deno.js'`)
         },
       },
       {

